servers/physics_3d/godot_soft_body_3d.cpp

```diff
@@ -595,7 +600,7 @@ void GodotSoftBody3D::generate_bending_constraints(int p_distance) {
 		const uint32_t adj_size = n * n;
 		unsigned *adj = memnew_arr(unsigned, adj_size);
 
-#define IDX(_x_, _y_) ((_y_) * n + (_x_))
+#define IDX(_x_, _y_) ((_y_)*n + (_x_))
```

Comment:
You might have an older clang-format version than the one used on CI / by `pre-commit`. So this change needs to be reverted to pass CI.

Reply:
Ah yes, I was about to update it. Thanks!